src/core/tracking.js

```diff
@@ -105,8 +105,23 @@ export function getAction(type) {
   }[type] || TRACKING_TYPE_INVALID;
 }
 
+export function doNotTrackEnabled({
```

Comment:
I think generally we're prefixing boolean vars and functions with with `is` so this could be `isDoNotTrackEnabled`  &lt;/nitpick&gt;